src/renderer/components/ft-channel-bubble/ft-channel-bubble.js

```diff
@@ -21,8 +21,20 @@ export default Vue.extend({
       selected: false
     }
   },
+  computed: {
+    sanitizedId: function() {
+      return 'channelBubble' + sanitizeForHtmlId(this.channelName)
+    }
+  },
   methods: {
-    handleClick: function () {
+    handleClick: function (event) {
+      if (event instanceof KeyboardEvent) {
+        if (event.target.getAttribute('role') === 'link' && event.key !== 'Enter') {
```

Comment:
What is this check for? the ft-channel-bubble component doesn't contain any elements with the link role

Reply:
Ah that makes sense.